src/measure_theory/prod.lean

```diff
@@ -333,6 +333,31 @@ by simp_rw [prod_apply (hs.prod ht), mk_preimage_prod_right_eq_if, measure_if,
   lintegral_indicator _ hs, lintegral_const, restrict_apply is_measurable.univ,
   univ_inter, mul_comm]
 
+local attribute [instance] nonempty_measurable_superset
+/-- If we don't assume measurability of `s` and `t`, we can bound the measure of their product. -/
+lemma prod_prod_le (s : set α) (t : set β) : μ.prod ν (s.prod t) ≤ μ s * ν t :=
+begin
+  by_cases hs0 : μ s = 0,
+  { rcases (exists_is_measurable_superset_of_measure_eq_zero hs0) with ⟨s', hs', h2s', h3s'⟩,
+    convert measure_mono (prod_mono hs' (subset_univ _)),
+    simp_rw [hs0, prod_prod h2s' is_measurable.univ, h3s', zero_mul] },
+  by_cases hti : ν t = ⊤,
+  { convert le_top, simp_rw [hti, ennreal.mul_top, hs0, if_false] },
+  rw [measure_eq_infi' μ],
+  simp_rw [ennreal.infi_mul hti], refine le_infi _, rintro ⟨s', h1s', h2s'⟩,
```

Comment:
I thought that in general we had only one command per line, unless this is a trivial one-liner closing a goal. Same thing below line 356.

Reply:
I am not aware of this style convention (I don't think it is in the style guide), but I'm happy to add some newlines.